libs/sr/ui/src/LightCone/LightConeInventory.tsx

```diff
@@ -15,6 +22,19 @@ export function LightConeInventory() {
   const { database } = useDatabaseContext()
   const [dirtyDatabase, setDirtyDatabase] = useForceUpdate()
 
+  // lives here until DB storage for UI sort/filter options is created
+  const initialSort = () => ({
+    sortType: lightConeSortKeys[0],
+    ascending: false,
+    rarity: [...allRarityKeys],
+    path: [...allPathKeys],
+  })
+
+  // no way to currently change state of sort until
+  // ability to store UI sort/filter options get added to DB storage
+  // until then, initialize a default sort
+  const { sortType, ascending } = initialSort()
```

Comment:
this should probably be a state later, once we do the buttons, but the scaffolding is good enough for now